### ISSUE TITLE
predict_images are all black or white

### ISSUE THREAD

**Issue by minastinisofking** [dali92002/DE-GAN#2]
Hello, DE-GAN author.I used watermark text dataset provided by you to train the model without changing any parameters in the model.But it seems doesn't work beacause the PSNR has always been below 1 and predict images are all black or white until I stop running in epoch 36.Can you please tell me where the problem is? Thank you!!!
![image](https://user-images.githubusercontent.com/54938769/100950892-61396180-3548-11eb-9053-a588d8487ec2.png)
![image](https://user-images.githubusercontent.com/54938769/100950947-79a97c00-3548-11eb-897f-9add02e3d526.png)

**Comment by dali92002** [comment]
Hello,

You can increase the weight of the binary_cross_entropy loss in line 209.
Make it 500 instead of 100 as the paper stating.

Or, Try to add some batch normalization layers in the generator,

Or, do them both,

Regards,

**Comment by gouravsinghbais** [comment]
Hi, 
I am having the same issue and I have tried adding batch normalization layers and used binary_cross_entropy loss as 500 but it is still not improving. Anything else that we could do to get the results.

Regards

**Comment by zzojin** [comment]
I come to same problem

**Comment by Ashwani-Dangwal** [comment]
Hey there, was anyone able to train the model using the code provided here?
